### PR TITLE
created set goals

### PR DESCRIPTION
**Pull request by kavishsanghvi**

### CODE REVIEW

File: api/goals.py [hunk 1]
```diff
@@ -61,4 +74,63 @@ def post(self):
             return make_response(jsonify(responseObject)), 401
 
 
+
+class CheckGoalsAPI(MethodView):
+    """
+    User Reservations
+    """
+    def post(self):
```
Comment (Viyeta) [inline api/goals.py:82]: Please use `@authorize` in this and the API above. Without it, you won't be able to access `g.user.id`. Also makes sure you import everything required for using the decorator.

**Comment by Viyeta** [comment]
Make sure to test the api on heroku after merging the PR

File: api/goals.py (hunk 1 above)
Comment (kavishsanghvi) [reply api/goals.py:84]: resolved

**Review by Viyeta — COMMENTED** [review]
We don't need `encode_auth_token` or `decode_auth_token` in anything other than `employee.py`. We just need to use `@authorize` to validate authenticated request.

**Comment by kavishsanghvi** [comment]
> We don't need `encode_auth_token` or `decode_auth_token` in anything other than `employee.py`. We just need to use `@authorize` to validate authenticated request.

I used the template so it already had these functions and I didn’t remove it.

**Comment by Viyeta** [comment]
> > We don't need `encode_auth_token` or `decode_auth_token` in anything other than `employee.py`. We just need to use `@authorize` to validate authenticated request.
> 
> I used the template so it already had these functions and I didn’t remove it.

Ok. You can remove it later. Since we use `@authorize` to authenticate requests, it's only needed in `data/employee.py`.

**Review by Viyeta — APPROVED** [review]
Looks good!

**Pull request merged**